src/model/Board.java

```diff
@@ -60,7 +59,10 @@ public void notifyListeners() {
 
 	public boolean placePlant(Plants plant, int row, int col) {
 		try {
-			if (this.tiles[row][col].getPlant() != null) return false;
+			if (this.tiles[row][col].getPlant() != null) {
+				System.out.println("Already a plant on that tile");
```

Comment:
User might not see console, is there another way to display this message?

Reply:
Oh i made a print out gui for it already. i could delete the systemprint in the next push.

Reply:
oh, you can leave it in, was just making a note of it since i didnt see the gui one